Lista_Studenti_2016.MD

```diff
@@ -25,7 +25,7 @@ Grupa: 1220F
 |Susanu Marius |https://github.com/susanu/ALF|https://github.com/susanu/SE|
 |Tila Simona|https://github.com/simonatila/ALF|https://github.com/simonatila/SE|
 |Tudor Georgiana|https://github.com/tudorgabriela/ALF|https://github.com/tudorgabriela/SE|
-|Ursachi Tudor|https://github.com/kylecrane||
+|Ursachi Tudor|https://github.com/TudorUrsachi9505/SE||https://github.com/TudorUrsachi9505/ALF
```

Comment:
1) Repository ALF then Repository SE
2) SE|| <-- 2x|, onlu 1x|
3) ALF <-- missing | at the end